macros/sql/get_tables_by_prefix_sql.sql

```diff
@@ -1,25 +1,25 @@
-{% macro get_tables_by_prefix_sql(schema, prefix, exclude='') %}
+{% macro get_tables_by_prefix_sql(schema, prefix, exclude='', database=target.database) %}
```

Comment:
I think this will work great, but there's an important thing to know about python (and therefore jinja, I assume?) argument defaults. Check out this article on [mutable default arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).

This code is A-OK because `target.database` is a string, which is immutable. It would only be a problem if the code read:
```
{% macro get_tables_by_prefix_sql(schema, prefix, exclude='', databases=[]) %}
```
or similar.